cloud-discord/cloud-jda/src/main/java/cloud/commandframework/jda/parsers/UserArgument.java

```diff
@@ -106,10 +112,16 @@ private UserArgument(
         NAME
     }
 
+    public enum Isolation {
+        GLOBAL,
+        GUILD
+    }
+
 
     public static final class Builder<C> extends CommandArgument.Builder<C, User> {
 
         private Set<ParserMode> modes = new HashSet<>();
+        private Isolation isolationLevel = Isolation.GUILD;
```

Comment:
I think the default, at least until 2.0 should be GLOBAL so we're not changing behaviour on a minor version

Reply:
My thinking was that users' privacy could be impacted by GLOBAL and so GUILD would be a better default option, but I understand your concern. I've changed the default to be GLOBAL in my newest changes.

Reply:
Yeah, I do agree. We can change the default for 2.0